Mathlib/Geometry/Manifold/Notation.lean

```diff
@@ -329,6 +333,47 @@ where
         | _ => return none
       | throwError "Couldn't find a `ModelWithCorners` with model space `{H}` in the local context."
     return m
+  /-- Attempt to find a model with corners on a space of continuous linear maps -/
+  -- TODO: should this also support continuous linear equivalences?
+  fromCLM : TermElabM Expr := do
+    match_expr e with
+    | ContinuousLinearMap k S _ _ _σ _E _ _ _F _ _  _ _ =>
+      if ← isDefEq k S then
+        -- TODO: check if σ is actually the identity!
+        let eK : Term ← Term.exprToSyntax k
+        let eT : Term ← Term.exprToSyntax e
+        let iTerm : Term := ← ``(𝓘($eK, $eT))
+        Term.elabTerm iTerm none
+      else
+        throwError "Coefficients {k} and {S} of {e} are not definitionally equal"
+    -- | ContinuousLinearEquiv k S _ _ _σ _σ' _ _ _E _ _ _F _ _ _ _ =>
+    --   if ← isDefEq k S then
+    --     -- TODO: check if σ is actually the identity!
+    --     let eK : Term ← Term.exprToSyntax k
+    --     let eT : Term ← Term.exprToSyntax e
+    --     let iTerm : Term := ← ``(𝓘($eK, $eT))
+    --     Term.elabTerm iTerm none
+    --   else
+    --     throwError "Coefficients {k} and {S} of {e} are not definitionally equal"
```

Comment:
Why is this code commented out? 👀

Reply:
Ah, okay! (I also finally saw the TODO above!)

I'm not sure about anything from the use case side, but seeing as it's almost identical to the branch for `ContinuousLinearMap`, it's not really adding any maintenance cost. I'd be inclined to support it! :)

Reply:
So was I, and then I turned on my brain and realised that this does not make mathematical sense. (Continuous linear equivalences are not an abelian group, with addition. I've removed the TODO and added a comment why they are not handled here. They could be handled in the future, as they are a manifold, but that can happen in a future PR.)